plugin/teksi_wastewater/interlis/interlis_model_mapping/interlis_exporter_to_intermediate_schema.py

```diff
@@ -2405,9 +2405,12 @@ def get_oid_prefix(self, oid_table):
         instance = self.tww_session.query(oid_table).filter(oid_table.active is True).first()
         if instance is None:
             logger.warning(
-                f'Could not find an active entry in table"{oid_table.__table__.schema}.{oid_table.__name__}". Setting to default instead.'
+                f'Could not find an active entry in table"{oid_table.__table__.schema}.{oid_table.__name__}". \
+                Returning an empty string, which will lead to Interlis Errors. \
+                Set the value \'active\' in table"{oid_table.__table__.schema}.{oid_table.__name__}" to \
```

Comment:
Set one value active that you want to use as prefix.

What happens if there is ore than one active?

Reply:
with more than one entry set to active, the export takes the first one. This is almost never the case because more than one active entry corrupts oid generation. Fix will be needed once we support multitenancy

Reply:
I don't want to support multiple active prefixes at the moment, but to make sure that there are not several set to active - I open a new issue for that https://github.com/teksi/wastewater/issues/210